CodingStandards/ProgramEntryPoint.cs

```diff
@@ -4,34 +4,50 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-
+/// <summary>
+/// Creates Program class to get current date and time to display appropriate message.
+/// </summary>
 class Program
 {
-        //This is the entry point for the program.
-        static int Main(string[] args)
-{
-    return writeMessage(dtNow, Console.Out);
-}
-
-        static DateTime dtNow = DateTime.Now;
-
-public static int writeMessage(DateTime CurrentDate, TextWriter writer)
-{
-    var intRetVal = 0; const Int32 MESSAGE_COUNT = 5; for (int MyIndex = 0; MyIndex < MESSAGE_COUNT; MyIndex++)
-    /*********************/
-    /* greater than 2016 */
-    /*********************/
-    if (CurrentDate.Year > 2016) writer.WriteLine("Happy New Year!");
-    else
-                writer.WriteLine("Hello From " + CurrentDate.Year);
-
-
-    //--------------------return the result value----------------------------
-
-
-
+    //This is the entry point for the program.
+    static DateTime currentDate = DateTime.Now;
+
+    /// <summary>
+    /// Calls the WriteMessage method and passes the dateTime variable.  
+    /// </summary>
+    /// <param name="args"></param>
+    /// <returns></returns>
+    static int Main(string[] args)
+    {
+        return WriteMessage(currentDate, Console.Out);
+    }
 
-    return intRetVal;
+    /// <summary>
+    /// Writes the appropriate message based on the dateTime variable. 
+    /// </summary>
+    /// <param name="currentDate"></param>
+    /// <param name="writer"></param>
+    /// <returns></returns>
+    public static int WriteMessage(DateTime currentDate, TextWriter writer)
+    {
+        var returnValue = 0;
+        const int messageCount = 5;
```

Comment:
Conventionally, const variables are placed at the top of a method with a space after them. This is not a coding standards violation but would be a good way to organize code.